tests/test_journal.py

```diff
@@ -431,6 +439,15 @@ def test_submission(self, journal, openreview_client, test_client, helpers):
         assert note.content['authorids']['value'] == ['~SomeFirstName_User1', 'andrewmc@mailfour.com']
         assert note.content['venue']['value'] == 'Withdrawn by Authors'
         assert note.content['venueid']['value'] == '.TMLR/Withdrawn_Submission'
+#         assert note.content['_bibtex']['value'] == '''@article{
+# userpaper,
+# title={Paper title {UPDATED}},
+# author={Test User and Andrew McCallum},
+# journal={Submitted to Transactions of Machine Learning Research},
+# year={2022},
+# url={https://openreview.net/forum?id=''' + note_id_1 + '''},
+# note={Under review}
+# }'''
```

Comment:
Should this be removed?

Reply:
I would like to keep it because there is bibtex entry for withdrawn invitations and I think we should, I need to confirm with Hugo which note  text to put.